src/components/OfferCard/OfferCard.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Heading from '../Heading';
+import StarRating from '../StarRating';
+import Box from '../Box';
+import Flex from '../Flex';
+import Text from '../Text';
+import Label from '../Label';
+import Tags from '../Tags';
+import flattenChildren from 'react-flatten-children';
+
+const hasOffers = (children) => {
+  const flattened = flattenChildren(children);
+  return flattened.filter(Boolean).length >= 1;
+};
+
+const OfferCard = (props) => {
+  return (
+    <>
+      <Box borderRadius="10px" backgroundColor="white">
+        <Flex flexDirection={['column-reverse', null, 'row']}>
+          <Box display="flex" flex="1" p={6} pb={[6, null, 0]}>
+            <Flex
+              flexDirection={['column', null, 'column']}
```

Comment:
```suggestion
              flexDirection="column"
```
If the styling is the same for all screens sizes, don't need a responsive array? applies for other similar stylings elsewhere